### ISSUE TITLE
No new versions

### ISSUE THREAD

**Issue by fsacer** [ropnop/impacket_static_binaries#6]
This seems to be out of date. Is there a way to make this build automatically from https://github.com/SecureAuthCorp/impacket repo once new version comes out?

**Comment by ropnop** [comment]
Yep, sorry about that. Looks like v0.9.20 is out on Impacket upstream, so I'll try to build and release binaries later tonight.

I would love a way to automatically trigger new builds and keep this repository up-to-date with upstream automatically, but I don't know a good way to do that. I would need to catch a notification or webhook from upstream when a new release is cut to trigger a new pipeline that builds this one, but I looked into it and doesn't look like it's possible. The best I can do for now is just check periodically when a new release is cut and do it manually - but i'm open for suggestions!

**Comment by ropnop** [comment]
v0.9.20 released :) https://github.com/ropnop/impacket_static_binaries/releases/tag/0.9.20-binaries

**Comment by fsacer** [comment]
Github seems to have a webhook that catches the release event https://help.github.com/en/actions/automating-your-workflow-with-github-actions/events-that-trigger-workflows#release-event-release and also you can get releases via API https://developer.github.com/v3/repos/releases/ also GitHub actions is free for public repos https://help.github.com/en/github/setting-up-and-managing-billing-and-payments-on-github/about-billing-for-github-actions

**Comment by fsacer** [comment]
but yeah listening to another repo seems to be problematic https://github.community/t5/GitHub-Actions/Triggering-by-other-repository/td-p/30668 but can be done

**Comment by ropnop** [comment]
Yeah, thats the part I got stuck on. Without getting Impacket's owner to set something up, it's not easy. I might play with a nightly pipeline that just queries the upstream every 24 hours or something. I'll keep exploring
